packages/mui-base/src/SliderUnstyled/SliderUnstyled.js

```diff
@@ -111,50 +110,83 @@ const SliderUnstyled = React.forwardRef(function SliderUnstyled(props, ref) {
 
   ownerState.marked = marks.length > 0 && marks.some((mark) => mark.label);
   ownerState.dragging = dragging;
+  ownerState.focusVisible = focusVisible;
```

Comment:
This may be misleading, as the `focusVisible` state in the Slider is the index of the focused thumb.

Reply:
It is misleading indeed. I'd rename it to `focusedThumbIndex`.